### PR TITLE
Indentation fix

### PR DESCRIPTION
**Pull request by siddharthgangwar25**
Fixed Indentation issue of  "if '__name__'=='__main__'" in many files from 30 days of Python

### CODE REVIEW

File: 30 Days of Code (Python)/D25-Running-Time-and-Complexity.py [hunk 1]
```diff
@@ -69,15 +69,15 @@
...
-    if _ == 1 or _ == 0:
-        print('Not prime')
-    else:
-        m=0
-        for a in range(2,int(math.sqrt(_))+1):
-            if _%a == 0:
-                m+=1
-        if m == 0:
-            print('Prime')
+    for _ in nums:
```
Comment (cclauss) [inline 30 Days of Code (Python)/D25-Running-Time-and-Complexity.py:72]: Can we please change `_` to `num`?  The single underscore character is reserved in Python for variables that we will not reuse but this variable is referred to on lines 73, 77, and 78.

File: 30 Days of Code (Python)/D25-Running-Time-and-Complexity.py [hunk 2]
```diff
@@ -69,15 +69,15 @@
...
+        if _ == 1 or _ == 0:
```
Comment (cclauss) [inline 30 Days of Code (Python)/D25-Running-Time-and-Complexity.py]:
```suggestion
        if _ in (0, 1):
```

File: 30 Days of Code (Python)/D28-RegEx,-Patterns,-and-Intro-to-Databases.py [hunk 3]
```diff
@@ -72,5 +72,5 @@
         if emailID.endswith('@gmail.com'):
             det[emailID] = firstName
 
-for _ in sorted(det.values()):
-    print(_)
\ No newline at end of file
+    for _ in sorted(det.values()):
+        print(_)
```
Comment (cclauss) [inline 30 Days of Code (Python)/D28-RegEx,-Patterns,-and-Intro-to-Databases.py]:
```suggestion
    for value in sorted(det.values()):
        print(value)
```

File: 30 Days of Code (Python)/D25-Running-Time-and-Complexity.py (hunk 1 above)
Comment (swapnanildutta) [reply 30 Days of Code (Python)/D25-Running-Time-and-Complexity.py:72]: That would be great.

**Review by swapnanildutta — APPROVED** [review]

**Pull request merged**